README.md

```diff
@@ -25,48 +26,45 @@ The prefered way to install HOMS is to use [Docker](https://www.docker.com/).
 
 ### In production
 
-1. Install [docker-compose](https://docs.docker.com/compose/install/).
-2. Download `docker-compose.yml`:
+1. Clone this repo and navigate to its directory:
 
     ```bash
-    wget https://raw.githubusercontent.com/latera/homs/master/docker-compose.yml
+    git clone git@github.com:latera/homs.git && cd homs
     ```
 
-3. For OS X users: make path to folder with HOMS shared in `Docker -> Preferences... -> File Sharing`.
+2. For OS X users: make path to folder with HOMS shared in `Docker -> Preferences... -> File Sharing`.
 
-4. Copy your (or default) `.env` file to your project's directory:
+3. Copy your (or default) `.env` file to your project's directory:
```

Comment:
It maybe not so obvious that you should copy `.env.sample` file from repository. I think other description should be set here or even `curl` should be used instead.